confidential-identity/wasm/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Has Mohammed reviewed the license text?

Reply:
@JesseALancaster was ok with Apache v2. But I am not sure about Mohammad, I'll contact him.

Reply:
It is ok to use the default apache v2